miscellaneous/eggnog_performance.md

```diff
@@ -15,4 +15,34 @@ The results are summarized in the table below:
 | 5               | 1                             | N.A. (43 finished)        | 21 min                       |
 | 10              | 1                             | N.A. (39 finished)        | 42 min                       |
 
-Each node has 3 hrs of wall time.
\ No newline at end of file
+(Each node has 3 hrs of wall time.)
+
+### Test with scratch drive (Recommended by NERSC)
+
+Copied the source files to `$SCRATCH` and ran the same tests. The results are summarized in the table below:
```

Comment:
Are these the same 50 genomes as above?

If you have the data, per genome size in GB, size in protein count, and run time would be interesting

Reply:
yea same genomes. I didn't calculate the genome info.

Reply:
> yea same genomes.

Can you make that explicit in the notes

Reply:
👍

Reply:
:+1: